frontend/src/components/Login/Login.jsx

```diff
@@ -56,23 +56,24 @@ const Login = () => {
               heures
             </p>
             <form onSubmit={login} id="login_form">
-              <label>
-                Adresse email :
-                {' '}
-                <input
-                  data-test-id="email-input"
-                  type="email"
-                  value={email}
-                  onChange={e => setEmail(e.target.value)}
-                />
-              </label>
-              {' '}
-              <Button
-                submit
-                data-test-id="email-button"
-              >
-                Recevoir le lien de connexion
-              </Button>
+              <Row alignItems="bottom">
+                <Col>
```

Comment:
l'affichage sur mobile est pas ouf
```suggestion
                <Col n="md-6 sm-12">
``` ?